numpy/core/src/multiarray/einsum.dispatch.c.src

```diff
@@ -56,6 +28,10 @@
  *             npy_ubyte, npy_ushort, npy_uint, npy_ulong, npy_ulonglong,
  *             npy_float, npy_float, npy_double, npy_longdouble,
  *             npy_float, npy_double, npy_longdouble#
+* #sfx  = s8, s16, s32, long, s64,
+ *        u8, u16, u32, ulong, u64,
```

Comment:
I'm not sure this is safe - there is no guarantee that these are the sizes of char, short, int, and long long.

Reply:
These suffixes aren't activated yet only f32 and f64, check `#NPYV_CHK#`